README.md

```diff
@@ -8,8 +8,16 @@ gem install wwwtxt
 
 ## Usage
 
+(Local)
+
+```
+bundle exec bin/wwwtxt -o "~/texts" get https://raw.githubusercontent.com/ryanmuller/www.txt/master/README.md
+```
+
+## Running tests
+
 ```
-wwwtxt get https://raw.githubusercontent.com/ryanmuller/www.txt/master/README.md
+ruby test/wwwtxt/test_page.rb
```

Comment:
This seems weird, however I'm not super super familiar with MiniTest (which I applaud you for trying out btw)